server.js

```diff
@@ -39,13 +39,23 @@ app.post('/api/v1/foods', function(request, response) {
 
 app.get('/api/v1/foods/:id', function (request, response) {
   var id = request.params.id
+
   Foods.find(id).then(function (data) {
       if (data.rowCount == 0) { return response.sendStatus(404) }
       response.json(data.rows[0])
     })
 
 })
 
+// PUT foods/:id
+app.put('/api/vi/foods/:id', function (request, response){
```

Comment:
we're not able to get this function to run (it seems) from the test file.

The `console.log()` on line 52 isn't being hit.